config/default/manager_auth_proxy_patch.yaml

```diff
@@ -17,6 +17,7 @@ spec:
                   operator: In
                   values:
                     - amd64
+                    - arm64
```

Comment:
Could you also update helm chart accordingly: https://github.com/kubernetes-sigs/cluster-api-operator/blob/main/config/chart/manager_auth_proxy_patch.yaml#L19